Docs: Mention yugabyte_upgrade user (#26724)
yugabyte/yugabyte-db: docs/content/preview/manage/ysql-major-upgrade-yugabyted.md

    If you have a pre-existing cluster, first upgrade it to the latest version in the v2024.2 series using the [upgrade instructions](../upgrade-deployment/).

> - If your cluster is configured with dedicated YB-Master nodes (i.e., no YB-TServer is running on the same node as the YB-Master), you must create a superuser named `yugabyte_upgrade` and add its credentials to the `.pgpass` file on each YB-Master node. This user can be safely removed once the upgrade process is complete.

Reviewer: ```suggestion
- If your cluster has dedicated master nodes (that is, nodes with YB-Master server only and no YB-TServer), you must create a superuser named `yugabyte_upgrade` and add its credentials to the `.pgpass` file on each YB-Master node. You can safely remove this user after the upgrade process is complete.
```
